spec/models/form526_submission_spec.rb

```diff
@@ -262,6 +262,17 @@
           exhausted_backup
         )
       end
+
+      it 'handles the edge case where an sub succeeds during query building' do
```

Comment:
Grammar typo: 'a', instead of 'an sub'. Also consider changing to 'submission', instead of 'sub'.